### ISSUE TITLE
Military Advantage/Disadvantage for Screen Reader Users

### ISSUE THREAD

**Issue by ironcross32** [pmotschmann/Evolve#896]
After reading about this info from the wiki, I was, for a time, unable to find it using my screen reader. I finally did. It's a tooltip that pops up when you mouse over the attack or launch campaign button. This is counter-intuitive for visually impaired people using screen readers, since we are often not focused on the mouse, but the keyboard. I am wondering if this information could be related in another way that is more intuitive to screen reader users?

I'm a bit pressed for time to think of something at the moment, but I'd be happy to think on it and provide ideas in the comments if this is something that could be done.

**Comment by Neurrone** [comment]
This applies not just to the advantage / disadvantage tooltips but all other instances where tooltips are used (for example over resource rates to view a breakdown of this info). Not having easy access to this information does make the play experience more difficult.

One way could be to use the existing approach where links are next to the item to read the tooltip.

**Comment by ironcross32** [comment]
This is true. W.R.T. putting links below each place that a mouseover exists, yes it certainly does work, but it's les than elegant. Still, if no other solution presents itself, this would be fine.

One issue that I have is that NVDA can be very spotty with its mouse jumping. Something that makes it a bit more reliable is to put Chrome in full screen, though this doesn't eliminate misjumps entirely.

There are plenty of good tidbits well worth knowing that are present in the mouseovers, and the only way to find out if something has a mouseover is to try to jump the mouse there. But, with this feature jumping to the wrong coordinates probably a third of the time, it makes things a bit more difficult.
